### DISCUSSION TITLE
cs_tools tools scriptability export no longer exports objects owned by tsadmin as of v1.5.6

### DISCUSSION THREAD

**Discussion by bmofkn** [thoughtspot/cs_tools#141] [Q&A] [answered]
We use `cs_tools tools scriptability export --include-types`  daily to bulk export the TML of all objects in our Org (passing in various types like `answer` or `liveboard`).  

Up until [v1.5.6](https://github.com/thoughtspot/cs_tools/tree/v1.5.6), this always included objects owned by tsadmin (the default Administrator account).  All of our "canned" liveboards and answers that are exposed to our TSE customers are owned by tsadmin, thus the need to export the objects daily for backup and recovery purposes.

As of v1.5.6, it seems all objects owned by tsadmin are skipped.  Is there a new parameter to pass into `cs_tools tools scriptability export --include-types` that will allow tsadmin objects to be exported?

Thanks!

**Comment by boonhapus** [comment]
🫠 #137 is the reason I changed this behavior @bmofkn . I will add an option to `scriptability export` like `--include-system-content` which will be off by default. Sorry to break your workflow! I was under the assumption that system owned content would break on import.

**Reply by boonhapus** [reply, marked as answer]
👉 #142

**Reply by bmofkn** [reply]
Thanks for the quick reply @boonhapus!  We have a long-standing process where we use the GraphQL `AssignChangeAuthor` API to make **tsadmin** the owner of our "official" TSE objects... perhaps we should re-think this and possibly create a **tse_admin** account to own these objects, so we don't conflate the two.  

Sincerelly appreciate that you are adding an option to include tsadmin objects.  You always deliver!

**Reply by boonhapus** [reply]
😂 thanks @bmofkn .. I would generally recommend to keep reliance on the system admin account as low as possible. We can do both options however, it's not a problem.
